### ISSUE TITLE
[nvim-treesitter] [0/74] Downloading...

### ISSUE THREAD

**Issue by zchrissirhcz** [optimizacija/neovim-config#5]
Hi, @optimizacija 

I just use this neovim-config for my neovim, it seems like the nvim-treesitter is something wrong, when I use nvim open a cpp file:

![图片](https://user-images.githubusercontent.com/3831847/144747649-d9fe9e84-b9b3-4646-88e7-7392d4e9d13d.png)

I've tried manually git clone the nvim-treesitter but it still show same error after reboot of the nvim. How can I do for this?

**Comment by zchrissirhcz** [comment]
The `:message` show that I'm not with a good network environment:

![图片](https://user-images.githubusercontent.com/3831847/144747754-8a600324-fba7-443e-a5bd-8d193b534df4.png)

But actually, other plugins can be git cloned.

**Comment by optimizacija** [comment]
@zchrissirhcz sorry for the late reply. I see that you have closed the PR after I did some fixes a couple of days ago. I hope the issue was fixed with the recent changes.

But while I'm on this topic... I know that this config breaks after a while (~couple of months). The problem is that a lot of the plugins that I'm using here are still quite new and they are introducing breaking changes more frequently.

I tend to use the most recent versions of installed plugins, but maybe I need to consider locking package versions (if that's possible).
